compiler/src/iree/compiler/Dialect/Stream/Transforms/test/refine_usage.mlir

```diff
@@ -82,14 +82,14 @@ util.func private @propagateBlocks(%cond: i1, %size: index) -> (!stream.resource
   // CHECK: %[[SELECT:.+]] = arith.select %[[COND]], %[[SPLAT1]], %[[FILL1]] : !stream.resource<external>
   %bb1_1_new = arith.select %cond, %splat1, %fill1 : !stream.resource<*>
   // CHECK: cf.cond_br %[[COND]], ^bb1(%[[FILL0]], %[[SELECT]]
-  // CHECK-SAME:               ^bb2(%[[FILL0]], %[[SELECT]]
+  // CHECK-SAME:               ^bb2
   cf.cond_br %cond, ^bb1(%fill0, %bb1_1_new : !stream.resource<*>, !stream.resource<*>),
                  ^bb2(%fill0, %bb1_1_new : !stream.resource<*>, !stream.resource<*>)
-// CHECK: ^bb2(%[[BB2_ARG0:.+]]: !stream.resource<transient>, %[[BB2_ARG1:.+]]: !stream.resource<external>)
+// CHECK: ^bb2
 ^bb2(%bb2_0: !stream.resource<*>, %bb2_1: !stream.resource<*>):
   // CHECK-NOT: stream.async.transfer
   %external_transfer = stream.async.transfer %bb2_1 : !stream.resource<*>{%size} -> !stream.resource<external>{%size}
-  // CHECK: util.return %[[BB2_ARG0]], %[[BB2_ARG1]] : !stream.resource<transient>, !stream.resource<external>
+  // CHECK: util.return %[[FILL0]], %[[SELECT]] : !stream.resource<transient>, !stream.resource<external>
```

Comment:
These changes are dropping some useful parts of the test?

Logs: https://github.com/iree-org/iree/actions/runs/9999003316/job/27639184068#step:5:1291